### DISCUSSION TITLE
How to set each segment must have at least 30s of duration?

### DISCUSSION THREAD

**Discussion by akmaldira** [SYSTRAN/faster-whisper#947] [Q&A]
<img width="605" alt="image" src="https://github.com/user-attachments/assets/64b48cb3-3d46-4dde-ab6f-fcb391dbd725">

I've try with with 
```code
vad_parameters={
    "min_speech_duration_ms": 30000
}
```

but didn't work

**Comment by MahmoudAshraf97** [comment]
use `without_timestamps=True` in transcription
